lib/mix/test/mix/tasks/xref_test.exs

```diff
@@ -933,6 +958,26 @@ defmodule Mix.Tasks.XrefTest do
         end
         """)
 
+        output =
+          capture_io(fn ->
+            assert Mix.Task.run("xref", ["graph", "--format", "json", "--output", "-"]) == :ok
+          end)
+
+        assert output ===
+                 String.trim_trailing("""
+                 {"lib/a.ex":{},"lib/b.ex":{}}
+                 """)
+
+        output =
```

Comment:
This is repeated from the above assertion.

Reply:
oh, good catch. Will fix.

Reply:
Ah, i remember, originally I made 'json' default to stdout, so there were two ways to express it. Then i changed it to match the 'dot' interface, and instead of removing the now redundant test I ended up with the test duplicated twice. I will push an update that removes the duplication.